### PR TITLE
tell Rack::Protection to set X-Frame-Options to SAMEORIGIN

### PR DESCRIPTION
**Pull request by tessi**
fixes https://www.openproject.org/issues/1719

### CODE REVIEW

**Comment by hschink** [comment]
Looks good (checked with [Clickjack script](http://www.cirt.net/clickjack-test)).

File: config.ru [hunk 1]
```diff
@@ -30,4 +30,5 @@
 
 require ::File.expand_path('../config/environment',  __FILE__)
 use Rack::Protection::JsonCsrf
+use Rack::Protection::FrameOptions
```
Comment (tilt) [inline config.ru:33]: @tessi / @meeee Have we defined the x-frame-option somewhere or what default is applied here?

Comment (tilt) [inline config.ru:33]: ok the title says SAMEORIGIN - but there seems to be a chrome issue

Comment (mfrister) [inline config.ru:33]: Yep, [default is sameorigin](https://github.com/rkh/rack-protection/blob/master/lib/rack/protection/frame_options.rb#L20).

What's your problem?

Comment (tilt) [inline config.ru:33]: On one of our instances Chrome fails rendering the response with

POST (canceled) multipart/form-data; boundary=----WebKitFormBoundaryEVlQUXeAtHB0jJU2

The same request is performed perfectly when triggered from the main frame

Comment (mfrister) [inline config.ru:33]: I don't know that problem. If you can't fix it, I guess we should just talk in person.

**Pull request merged**